shared/profile/render.native.js

```diff
@@ -121,51 +123,82 @@ class Render extends Component<void, Props, State> {
         <Box style={{...styleHeader, backgroundColor: trackerStateColors.header.background}}>
           {this.props.onBack && <BackButton title={null} onClick={this.props.onBack} style={{marginLeft: 16}} iconStyle={{color: globalColors.white}} />}
         </Box>
-        <ScrollView style={{flex: 1, backgroundColor: trackerStateColors.header.background}} contentContainerStyle={{backgroundColor: globalColors.white}}>
+        <ScrollView style={{flex: 1, backgroundColor: globalColors.white}} contentContainerStyle={{backgroundColor: globalColors.white}}>
           {proofNotice && (
             <Box style={{...styleProofNotice, backgroundColor: trackerStateColors.header.background}}>
               <Text type='BodySmallSemibold' style={{color: globalColors.white}}>{proofNotice}</Text>
             </Box>
           )}
-          <UserBio
-            type='Profile'
-            avatarSize={AVATAR_SIZE}
-            loading={this.props.loading}
-            username={this.props.username}
-            userInfo={this.props.userInfo}
-            currentlyFollowing={this.props.currentlyFollowing}
-            trackerState={this.props.trackerState}
-          />
-          <UserActions
-            style={styleActions}
-            trackerState={this.props.trackerState}
-            currentlyFollowing={this.props.currentlyFollowing}
-            onFollow={this.props.onFollow}
-            onUnfollow={this.props.onUnfollow}
-            onAcceptProofs={this.props.onAcceptProofs}
-          />
+          <Box style={{...globalStyles.flexBoxColumn, position: 'relative'}}>
+            <Box style={{position: 'absolute', backgroundColor: trackerStateColors.header.background, top: 0, left: 0, right: 0, height: 56}} />
+            <LoadingWrapper
+              style={{minHeight: 220}}
+              duration={500}
+              loading={this.props.loading}
+              loadingComponent={(
+                <UserBio
```

Comment:
does this really need to be duplicated? can't doneLoadingComponent = loadingComponent if its not specified or something?